tests/test_wheel.py

```diff
@@ -42,3 +42,13 @@ def test_choose_returns_random_bin_object_with_seed(self):
         randomly_selected_bin_object = self.wheel.choose()
 
         self.assertIn(self.oc1, randomly_selected_bin_object)
+
+    def test_get_outcome_returns_outcome_object(self):
+        self.wheel.addOutcome(1, self.oc1)
+        get_outcome_result = self.wheel.getOutcome(self.oc1.name)
+        expected_get_outcome_result = self.oc1
```

Comment:
verbose. let's rename to `outcome` and `expected_outcome`

Reply:
Done.

Reply:
still there is `getOutcome_result` variable for which pylint reports. better change this to `outcome`.

Reply:
also please address the "Consider iterating with .items() (consider-using-dict-items)" issue reported by pylint. this was already suggested in a comment.